internal/cluster/distribution/eks/service.go

```diff
@@ -186,6 +186,10 @@ const (
 	// are being provisioned.
 	NodePoolStatusCreating NodePoolStatus = "CREATING"
 
+	// NodePoolStatusDeleted is the status used when the node pool resources
+	// are aleready removed.
```

Comment:
```suggestion
	// are already removed.
```

Reply:
[Fixed](https://github.com/banzaicloud/pipeline/compare/5c1d744e7fc86f9e1613cc1df3342d345d33ca4a..564220550d148e21191fff10185a77b4f446caf9).

The funny thing is, I'm using a spell checker and it didn't go off, checked my custom dictionary and I don't have this word in it, so I checked the internet, but couldn't find any definition, weird.